forms.rst

```diff
@@ -702,6 +702,16 @@ the choice is ultimately up to you.
 
         $form->get('dueDate')->setData(new \DateTime());
 
+Rendering the Same Form Multiple Times on the Same Page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In some situations you might find yourself in need of having the same form on
+the same page multiple times. By default, a form name is based off the class of
+the form type. To differentiate between the forms, you can create a form using
+:method:`FormFactoryInterface::createNamed() <Symfony\\Component\\Form\\FormFactoryInterface>`.
+Doing so, will change the html name of the form and Symfony will be able to see
+which of the forms was submitted, and handle it accordingly.
```

Comment:
the `createNamed` is currently 100% undocumented while there are dozens of questions about it on stack overflow, hence I'd like to see it in the docs.

Personally I would not really use this but put everything in 1 form and add a collection, but it seems like others do use it.